packages/docs-page/content-api/index.ts

```diff
@@ -7,11 +7,28 @@ const DEFAULT_HEADERS = {
   },
 }
 
+// Courtesy helper for warning about missing env vars during development
+const checkEnvVarsInDev = () => {
+  if (process.env.NODE_ENV === 'development') {
+    if (!MKTG_CONTENT_API || !MKTG_CONTENT_API_TOKEN) {
+      const message = [
+        'Missing environment variables required to fetch remote content:',
+        MKTG_CONTENT_API ? '\t- `MKTG_CONTENT_API`' : false, 
+        MKTG_CONTENT_API_TOKEN ? '\t- `MKTG_CONTENT_API_TOKEN`' : false,
```

Comment:
Oops I ninja edited....
```suggestion
        !MKTG_CONTENT_API ? '  - `MKTG_CONTENT_API`' : false, 
        !MKTG_CONTENT_API_TOKEN ? '  - `MKTG_CONTENT_API_TOKEN`' : false,
```